packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -134,7 +134,10 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
         ...divProps
         }
         {
-        ...rootProps
+        // root props has been deprecated and should get removed.
```

Comment:
Add change file.

Is it possible to add a test to cover the issue you were seeing?

Reply:
+1 on the need for a test

---
In reply to: [176491968](https://github.com/OfficeDev/office-ui-fabric-react/pull/4335#discussion_r176491968) [](ancestors = 176491968)

Reply:
The break was a typescript break, this would not break functionality. And since webpack build caught the problem I would say there is a problem in the build since it let it through, while local npm start caught the type error. I'm not sure adding a test is correct here.

Reply:
@manishgarg1 this really doesn't need a test, can you approve?